test/new_tests/test_is_connected.py

```diff
@@ -35,10 +35,11 @@ def _connect(self):
 
```

Comment:
General comment for the tests. Have the tests been run against a server with auth enabled? That's when username and password matters. If connect(user, pass) no longer works and user, pass need to be in the client config then some tests might be broken. (any that rely on connect(user, pass))